packages/mandelbrot/src/filters.js

```diff
@@ -10,13 +10,13 @@ module.exports = function (theme, env, app) {
             if (!item.path) {
                 return '/';
             }
-            return theme.urlFromRoute('page', { path: item.path });
+            return theme.urlFromRoute('page', { path: item.path }).replace(/:/g, '%3A');
```

Comment:
The `theme.urlFromRoute` function is used elsewhere as well. Would it maybe make more sense to add that replacing inside that function? There's already a cleaning function there that may be the proper place: https://github.com/frctl/fractal/blob/main/packages/web/src/theme.js#L147-L149

Reply:
I tried this but `urlFromRoute` is also used when generating file names and so produces HTML files with `%3A` in their names. I'm not discounting a better place for this `replace` though, I'm just not sure where would be better.

Reply:
By "elsewhere" I did not mean other themes, rather that it's used directly in some nunjucks templates as well. If that was to be added everywhere, that's a lot of duplication TBH.

I can take a look sometime, maybe I can come up with some ideas on how to bypass the filename issue.

Reply:
Ack on further testing this doesn't work while the server is running, it fouls up the ajax page navigation. Going to rethink!